config/purposes/scrna_core_cell_extraction.wip.yml

```diff
@@ -86,14 +86,20 @@ LRC Bank Seq:
   :asset_type: tube
   :target: SampleTube
   :type: Tube::Purpose
-  :creator_class: LabwareCreators::PooledTubesBySample
+  :creator_class:
+    name: LabwareCreators::PlateSplitToTubeRacks
+    args: &fluidx_tube_creation_config
+      child_seq_tube_purpose_name: LRC Bank Seq
+      child_spare_tube_purpose_name: LRC Bank Spare
   :ancestor_stock_tube_purpose_name: LRC Blood Vac
```

Comment:
Just wondering if we might want this to be the LRC Blood Aliquot... that will be the one that they go back to to repeat isolations from, and _think_ it'll be the one the requests come out of.

But then, the Vac tube barcode is the one that'll be in the Supplier Name - so this is probably correct 😅

Reply:
I think it's ok, the ultimate ancestor is always the Vac tube.
They definitely only go back to the Blood Aliquot tube, there is enough volume left there? The Vac has gone to another lab and is not accessible?
I'm using the barcode in the tube 'pool name' currently.
We may need to understand better how they plan to tell all these tubes apart given we aren't printing labels on them anyway.

Reply:
Yep think the Vac tube will have gone to faculty by this point, so it will be going back to the aliquot tube. 
I guess I assumed they would print barcodes for the aliquot tubes - but maybe they're FluidX ones - will check with Lesley.

Reply:
Yep Lesley confirmed they'll print barcodes for the aliquot tubes